test/Altinn.Notifications.IntegrationTests/Notifications.Integrations/TestingConsumers/EmailStatusConsumerTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Altinn.Notifications.Core.Enums;
+using Altinn.Notifications.Core.Models.Notification;
+using Altinn.Notifications.Core.Models.Orders;
+using Altinn.Notifications.Integrations.Kafka.Consumers;
+using Altinn.Notifications.IntegrationTests.Utils;
+
+using Microsoft.Extensions.Hosting;
+
+using Xunit;
+
+namespace Altinn.Notifications.IntegrationTests.Notifications.Core.Consumers;
+
+public class EmailStatusConsumerTests : IDisposable
+{
+    private readonly string _statusUpdatedTopicName = Guid.NewGuid().ToString();
+    private readonly string _sendersRef = $"ref-{Guid.NewGuid()}";
+
+    [Fact]
+    public async Task RunTask_ConfirmExpectedSideEffects()
+    {
+        // Arrange
+        Dictionary<string, string> vars = new()
+        {
+            { "KafkaSettings__EmailStatusUpdatedTopicName", _statusUpdatedTopicName },
+            { "KafkaSettings__Admin__TopicList", $"[\"{_statusUpdatedTopicName}\"]" }
+        };
+
+        using EmailStatusConsumer consumerService = (EmailStatusConsumer)ServiceUtil
+                                                    .GetServices(new List<Type>() { typeof(IHostedService) }, vars)
+                                                    .First(s => s.GetType() == typeof(EmailStatusConsumer))!;
+
+        (NotificationOrder Order, EmailNotification Notification) = await PostgreUtil.PopulateDBWithOrderAndEmailNotification(_sendersRef);
```

Comment:
We can discard the values not in use.
```suggestion
        _ = await PostgreUtil.PopulateDBWithOrderAndEmailNotification(_sendersRef);
```